client/src/pages/Info/styles.module.scss

```diff
@@ -0,0 +1,6 @@
+.main {
+  height: calc(100vh - 150px);
+  width: 100%;
+  padding: 20px;
+  text-align: center;
+}
```

Comment:
please move this selector to shared styles file to avoid duplicating. or you can create separate shared component `<Main/>` that will be just a styled wrapper for and place styles inside it